introduction_to_amazon_algorithms/linear_learner_abalone/Linear_Learner_Regression_csv_format.ipynb

```diff
@@ -13,15 +13,16 @@
     "## Contents\n",
```

Comment:
<ol><li>s/let us/Let us.</li><li>It might be easier to read the paragraph if it used markdown to format code like <code>sagemaker.session.s3_input</code>,  <code>content_type</code>,  <code>text/csv</code>  etc.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done.</p><p> </p>